packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/Abs_L1TokenGateway.sol

```diff
@@ -148,11 +156,15 @@ abstract contract Abs_L1TokenGateway is iOVM_L1TokenGateway, OVM_CrossDomainEnab
      * @param _from Account to pull the deposit from on L1
      * @param _to Account to give the deposit to on L2
      * @param _amount Amount of the ERC20 to deposit.
+     * @param _data Data to forward to L2.
+     * @param _l2Gas Gas limit for the provided message.
      */
     function _initiateDeposit(
         address _from,
         address _to,
-        uint _amount
+        uint _amount,
+        bytes calldata _data,
+        uint32 _l2Gas
```

Comment:
Can we move this declaration before the `bytes data`? It's just for trivial purposes really, mostly to make it easier to look at values when decoding transactions, rather than scroll past a potentially very large data field. Not too important but helps with my ocd :)

Reply:
Will do, I agree that's a nicer feeling.
Also... I'm kind of suprised I'm using `uint` instead of `uint256`...

Reply:
yes, I like to stick to explicit declarations rather than rely on defaults. There was even a point in the compiler where a function signature with uint did not generate the same hash as a uint256 !